src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -50,16 +50,15 @@ class ScalafixPlugin implements Plugin<Project> {
 
         project.tasks.named('check').configure { it.dependsOn checkTask }
 
+        if (!extension.autoConfigureSemanticdb) { // TODO remove in a future version
+            logger.warn("WARNING: the 'autoConfigureSemanticdb' property is deprecated. Please use 'semanticdb.autoconfigure' instead")
+        }
```

Comment:
this message was being printed once for every source set. I moved it out of the loop